docs/_includes/forkme_banner.html

```diff
@@ -1 +1 @@
-<a class="forkme_banner" href="{{ site.github.repository_url }}">View on GitHub <i class="fab fa-github"></i></a>
+<a class="forkme_banner" href="{{ site.contact.github.project-link }}">View on GitHub <i class="fab fa-github"></i></a>
```

Comment:
is it possible to make this not disappear when scrolling down the page? always visible means that potential developers will more likely click on it.

Reply:
Ah, this was not intended that the banner disappears. I will fix this.